### ISSUE TITLE
Bug: Error being shown all over the screen instead only in the container div

### ISSUE THREAD

**Issue by SandeepGumaste** [safe-global/web-core#1643]
## Bug description
Error being shown all over the screen instead only in the container div
## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Click on 'Open $Safe Claiming app' in dev

## Screenshots
![image](https://user-images.githubusercontent.com/73634237/216812852-98f5ea80-b02e-4a6a-92eb-730f4b41dbb1.png)

**Comment by SandeepGumaste** [comment]
@iamacook @katspaugh  Can I fix this?

**Comment by iamacook** [comment]
> @iamacook @katspaugh Can I fix this?

@SandeepGumaste, thank you for reporting the bug! You're more than welcome to look into this.

It looks like it's related to the [claiming app](https://github.com/safe-global/safe-react-apps/tree/development/apps/safe-claiming-app) directly but I am not sure so I won't transfer the issue. @schmanu, can you shed any light on this?

**Comment by schmanu** [comment]
> It looks like it's related to the [claiming app](https://github.com/safe-global/safe-react-apps/tree/development/apps/safe-claiming-app) directly but I am not sure so I won't transfer the issue. @schmanu, can you shed any light on this?

Yes it is related to the claiming app which is embedded as an iframe here and not web-core directly.
